modules/porous_flow/src/utils/PorousFlowVanGenuchten.C

```diff
@@ -203,4 +203,221 @@ d2RelativePermeabilityNW(Real seff, Real m)
 
   return -0.25 * std::pow(seff, -1.5) * b + std::pow(seff, -0.5) * db + std::sqrt(seff) * d2b;
 }
+
+Real
+capillaryPressureHys(Real sl,
+                     Real slmin,
+                     Real sgrdel,
+                     Real alpha,
+                     Real n,
+                     const LowCapillaryPressureExtension & low_ext,
+                     const HighCapillaryPressureExtension & high_ext)
+{
+  if (sl < low_ext.S) // important for initializing low_ext that this is < and not <=
```

Comment:
All these return values makes this confusing. Would it be better to make this an if, else if, else function and define the output up front.

```
Real output;
if (sl < low_ext.S)
  output = ...
else if (sl >high_ext.S)
  output = ...
else
  output = ...
return output;

Reply:
done